### DISCUSSION TITLE
We have 10+ Stations. We can list all 10 on Xiph YP, but only one is listed (rotating) on Internet-Radio

### DISCUSSION THREAD

**Discussion by cooper-fm** [AzuraCast/AzuraCast#7309] [Q&A]
Hi,

As the title states, we have a growing radio station network with 10+ stations at the moment. There's a weird bug I'm trying to fix.

First station added - listed on Xiph and Internet-Radio (IR) correctly.
Adds more stations within AzuraCast (AZ) panel, cloning from the original station, including a clone of the mount points.
The last station added typically replaces the current station listed on IR.

Nothing particularly weird about our setup, stations are added with SSL native support. Links are typically:

https://radio.hearme.fm:8000/stream
https://radio.hearme.fm:8010/stream

It might also be worthwhile pointing out that we're currently migrating from Centova to AZ, and the problem didn't happen on our bare metal server with Xiph or IR. I'm thinking it may have something to do with the docker system? But this is above my pay-grade. If it has got something to do with docker, I'll have to get my chief administrator involved.

Thanks guys

**Comment by BusterNeece** [comment]
It's possible that Internet-Radio thinks your streams are all the same stream on account of coming from the same base domain and only slightly varying ports, but I don't know how they determine this under the hood.

You mentioned that you're cloning the same station over and over to generate the new ones, and this shouldn't cause a problem, provided you reload/restart the configuration on the new station once it's set up.

I strongly doubt it's a Docker-related issue at all, but without any further info, I don't know what's causing it on I-R's side.

**Reply by cooper-fm** [reply]
Hi,

Thanks for the response. The setup is pretty much exactly the same as what I had on our bare metal server. 

All stations are being reloaded, but I've even tried just creating a new station and still the problem is there.

Spoken to my tech team and they're not too familiar with the whole YP process
